File: apprefurbished/src/main/java/com/androidcollider/easyfin/managers/chart/setup/ChartSetupModule.java
package com.androidcollider.easyfin.managers.chart.setup;

import android.content.Context;

import dagger.Module;
import dagger.Provides;

/**
 * @author Ihor Bilous
 */

@Module
public class ChartSetupModule {

    @Provides
    ChartSetupManager provideChartSetupManager(Context context) {
        return new ChartSetupManager(context);
    }
}

File: apprefurbished/src/main/java/com/androidcollider/easyfin/common/managers/chart/setup/ChartSetupModule.kt
package com.androidcollider.easyfin.common.managers.chart.setup

import com.androidcollider.easyfin.common.managers.resources.ResourcesManager
import dagger.Module
import dagger.Provides

/**
 * @author Ihor Bilous
 */
@Module
class ChartSetupModule {
    @Provides
    fun provideChartSetupManager(resourcesManager: ResourcesManager): ChartSetupManager {
        return ChartSetupManager(resourcesManager)
    }
}